lib/src/child-writer/writer-proxy.js

```diff
@@ -197,18 +165,23 @@ return isElevated().then((elevated) => {
   console.log('Re-spawning with elevation');
 
   return new Bluebird((resolve, reject) => {
-    const child = childProcess.spawn(EXECUTABLE, ETCHER_ARGUMENTS);
-
-    child.stdout.on('data', (data) => {
-      console.log(data.toString());
-    });
-
-    child.stderr.on('data', (data) => {
-      console.error(data.toString());
+    ipc.config.id = process.env.IPC_CLIENT_ID;
+    ipc.config.silent = true;
+    ipc.connectTo(process.env.IPC_SERVER_ID, () => {
+      ipc.of[process.env.IPC_SERVER_ID].on('error', reject);
+      ipc.of[process.env.IPC_SERVER_ID].on('connect', () => {
+        const child = childProcess.spawn(EXECUTABLE, ETCHER_ARGUMENTS);
+        child.on('error', reject);
+        child.on('close', resolve);
+
+        const emitMessage = (data) => {
+          ipc.of[process.env.IPC_SERVER_ID].emit('message', data.toString());
+        };
+
+        child.stdout.on('data', emitMessage);
+        child.stderr.on('data', emitMessage);
```

Comment:
I guess this means that the server can't tell whether the 'message' has come from stdout or stderr of the child? Would it be worth setting up two IPC channels, so the server can deal with errors different to normal output?